xtask/src/main.rs

```diff
@@ -105,10 +105,10 @@ impl Config {
 #[derive(Debug, Deserialize)]
 struct GithubConfig {
     /// The username to use for authentication.
-    user: String,
+    user: Arc<str>,
 
     /// The personal access token to use for authentication.
-    token: String,
+    token: Arc<str>,
```

Comment:
This seems like an independent and unnecessary change. Why use `Arc<str>` here?

Reply:
To avoid moving the `String` around. Since the contents are never mutated, this should functionally only be a benefit as it implements `AsRef<str>` just like `String`.

But at the same time, it's also totally fair if this to too much fluff; it's not exactly a huge deal to copy a few strings a few times a release tool :) I just noticed it and figured that, since the previous implementations didn't need to copy, I'd make sure the new one didn't either.

Reply:
Only [once](https://github.com/ruma/ruma/blob/82a6cb5e9ee770f26e36d19774cbca23e55d8cc8/xtask/src/release.rs#L271). I'll just undo the change. :)